File: Leetcode/fizz-buzz.cpp
// https://leetcode.com/explore/interview/card/top-interview-questions-easy/102/math/743/
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string>solution;
        for(int i = 1; i<=n; i++)
        {
            if(i%3 ==0 && i%5 ==0)
            {
                solution.push_back("FizzBuzz");
                continue;
            }    
            if(i%3 == 0)
                solution.push_back("Fizz");
            else if(i%5 == 0)
                solution.push_back("Buzz");
            else
                solution.push_back(to_string(i));
                
        }
        return solution; 
    }
};

File: Leetcode/fizz_buzz.py
# https://leetcode.com/problems/fizz-buzz/submissions/1907261972/
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        result = [] 
        for index in range(1, n+1):
            if index% 3==0 and index %5==0:
                result.append("FizzBuzz")
            elif index % 3 == 0:
                result.append("Fizz")
            elif index % 5 == 0:
                result.append("Buzz")  
            else:
                result.append(str(index)) 
        return result
